out/out_command.go

```diff
@@ -55,18 +55,24 @@ func (command *OutCommand) Run(sourceDir string, request OutRequest) (OutRespons
 
 	bucketName := request.Source.Bucket
 
-	acl := "private"
+	options := s3resource.NewUploadFileOptions()
+
 	if request.Params.Acl != "" {
-		acl = request.Params.Acl
+		options.Acl = request.Params.Acl
+	}
+
+	if request.Params.ContentType != "" {
```

Comment:
Emptiness check here is redundant since it defaults to blank anyway. Looks like it was used above since it otherwise defaults the ACL to `private`, which makes sense and justifies the constructor as well.